src/DynamicRupture/FrictionLaws/LinearSlipWeakening.cpp

```diff
@@ -1,6 +1,13 @@
 #include "LinearSlipWeakening.h"
 namespace seissol::dr::friction_law {
 
+void NoSpecialization::resampleSlipRates(
+    real (&resampledSlipRate)[dr::misc::numPaddedPoints],
+    real const (&slipRateMagnitude)[dr::misc::numPaddedPoints]) {
+  resampleKrnl.originalQ = slipRateMagnitude;
+  resampleKrnl.resampledQ = resampledSlipRate;
+  resampleKrnl.execute();
+}
```

Comment:
Why did you move it here?

Reply:
`FL=6` only works nicely without resampling. Otherwise, we get some artefacts in the solution (totally unrelated to #604). So I need `FL=6` without resampling and `FL=16` with resampling, so I decided to move the resampling to the specialization.
`NoSpecialization`: leaves the strength as it was, and resamples the slip rate.
`BiMaterialFault`: regularizes the strength and leaves the slip rate as it was.
Awful, I know :D

Reply:
That's strange. It might be related to negative SR induced by the resampling combined with the abs(SR) in the PC routine.